README.md

```diff
@@ -427,8 +396,7 @@ end
 ```
 
 We have defined a constant `Outcome` in the class of type `Struct`. All outcome
-values are `Struct`s so they are uniformly structured. A `Struct` is essentially
-a list of named values.
+values are `Struct`s. A `Struct` is essentially a list of named values.
```

Comment:
Realize the description did not change here, but "like a list" is not how I think of Structs. Maybe hash, dictionary, another analogy.